src/node.cc

```diff
@@ -4556,7 +4557,9 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
   {
     SealHandleScope seal(isolate);
     bool more;
+    PERFORMANCE_MARK(&env, LOOP_START);
     do {
+      PERFORMANCE_MARK(&env, LOOP_FRAME);
```

Comment:
What is `LOOP_FRAME` expected to measure? Now that we don't need to separately flush the v8 microtask queue we do not return to the body of this loop until the event loop has run to completion.